WcaOnRails/config/locales/kaminari.en.yml

```diff
@@ -0,0 +1,19 @@
+# Sample localization file for Kaminari. You can override these values in your app's locales file if you want.
+
+en:
+  views:
+    pagination:
+      first: "&laquo; First"
+      last: "Last &raquo;"
+      previous: "&lsaquo; Prev"
+      next: "Next &rsaquo;"
+      truncate: "&hellip;"
+  helpers:
+    page_entries_info:
+      one_page:
+        display_entries:
+          zero: "No %{entry_name} found"
+          one: "Displaying <b>1</b> %{entry_name}"
```

Comment:
Yeah, you're right, see [this](https://github.com/amatsuda/kaminari/blob/0fcfad2030c3558471f6136339a06b87c3d888f2/lib/kaminari/helpers/action_view_extension.rb#L102)